README.md

```diff
@@ -94,136 +108,135 @@ If you prefer not to use any of the aforementioned dependency managers, you can
-
-    ```swift
-    StitchClientFactory
-        .create(appId: "<your-client-app-id>")
-        .done { (client: StitchClient) in
-            // Perform requests to Stitch using the variable "client",
-            // or assign the value of the variable "client" to some
-            // property accessible outside this closure.
+## Example Usage
 
-            // For example, if this is in a class which has a
-            // stored StitchClient property named "stitchClient":
-            self.stitchClient = client
-        }.cauterize()
-    ```
+### Creating a new app with the SDK (iOS)
 
-    This will only instantiate a client but will not make any outgoing connection to Stitch.
+#### Set up an application on Stitch
+1. Go to [https://stitch.mongodb.com/](https://stitch.mongodb.com/) and log in to MongoDB Atlas.
+2. Create a new app in your project with your desired name.
+3. Go to your app in Stitch via Atlas by clicking Stitch Apps in the left side pane and clicking your app.
+3. Copy your app's client app id by going to Clients on the left side pane and clicking copy on the App ID section.
+4. Go to Providers from Users in the left side pane and edit and enable "Allow users to log in anonymously".
 
-2. For guidance on how to perform this initialization cleanly in the context of developing an iOS app, see the page [Initialize StitchClient](https://docs.mongodb.com/stitch/getting-started/init-stitchclient/#ios-sdk) in the MongoDB Stitch documentation.
+#### Set up a project in XCode/CocoaPods using Stitch
 
-3. Since we enabled anonymous log in, let's log in with it; add the following after you've initialized your new `StitchClient`:
-
-	```swift
-	self.stitchClient.fetchAuthProviders().then { (authProviderInfo: AuthProviderInfo) in
-            if (authProviderInfo.anonymousAuthProviderInfo != nil) {
-                print("logging in anonymously")
-                return self.stitchClient.anonymousAuth()
-            } else {
-                print("no anonymous provider")
-            }
-        }.then { (userId: String) in
-            print("logged in anonymously as user \(userId)")
-        }.catch { error in
-            print("failed to log in anonymously: \(error)")
-        }
-	```
+1. Download and install [XCode](https://developer.apple.com/xcode/)
+2. Create a new app project with your desired name. Ensure that Swift is the selected language.
+3. Navigate to the directory of the project in a command line, and run `pod init`.
+4. In the `Podfile` that is generated, add the following line under the dependencies for your app target:
 
-4. Now run your app in XCode by going to product, Run (or hitting ⌘R).
-5. Once the app is running, open up the Debug Area by going to View, Debug Area, Show Debug Area.
-6. You should see log messages like:
+```ruby
+    pod 'TODO_FIXME_StitchSDK', '~> 4.0.0'
```

Comment:
[update] This can be `pod 'StitchSDK', '~> 4.0.0'`. We should write a line that explains the services as well. I can do that part in my PR